pkgs/development/python-modules/yq/default.nix

```diff
@@ -27,6 +27,7 @@ buildPythonPackage rec {
     pyyaml
     xmltodict
     argcomplete
+    jq
```

Comment:
if the package is "shelling" out to it, this should be patched in the source to do so. Otherwise there's no guarantee that it will work when imported as a python module.

Reply:
Thanks for pointing that out! I patched the package; could you please take another look?